rep-2005.rst

```diff
@@ -39,20 +39,53 @@ The list below is curated to include repositories that are:
 
 We aim that all of these repositories are also:
 
-* actively maintained and following accepted software development processes, including code review and testing.
+* actively maintained and following accepted software development processes, including code review and testing; and
+* at Quality Level 3 or better, as defined in REP 2004; and
+* are maintained by an organization or at least two individuals.
+
+The list should be self-contained and exhaustive: if a package appears on the list, then all of its ROS 2 dependencies must also appear on the list (third-party or system dependencies should not be included).
 
 
 Change process
 ==============
 
 This content is maintained at GitHub in the [REP repo](https://github.com/ros-infrastructure/rep).
-To propose changes, such as adding or removing items, make a pull request to that repo.
-The ensuing public review and discussion will evaluate the change in light of the purpose and scope explained above, with the final merge decision being made by one of the REP maintainers.
+To propose changes, such as adding or removing items, make a pull request (PR) to that repo.
+The ensuing public review and discussion will evaluate the change in light of the purpose and scope explained above.
+The final merge decision on a given PR will be made by the ROS 2 Technical Steering Committee (TSC) by a simple voting process:
+
+* Each TSC member organization may vote on a PR by commenting with a score according to the [REP voting guidelines](https://www.ros.org/reps/rep-0010.html#voting-scores).
+* A TSC member may update their vote (e.g., following changes or discussion) with a new comment while the PR remains under review (i.e., before it has been merged); the new vote takes the place of the old one.
+* The PR will be merged if and when the sum of TSC member vote values is **>= 3** (votes of `+0` and `-0` are treated equally as 0 when computing the sum).
+* The PR will be declined if it becomes numerically impossible based on existing votes to meet the minimum vote threshold.
+* The proposer will be encouraged to withdraw a PR if after a significant amount the time the minimum vote threshold is still not met.
+
+For example, if a PR garners the following 6 votes from TSC members: `+1`, `+1`, `-1`, `+0`, `+1`, `+1`, then the sum would be 3 and the PR would be merged.
+
+The minimum vote threshold necessary to merge a PR may be changed by a majority vote of the TSC.
```

Comment:
With the sentence above, that all dependencies are added to this list as well, I would like to stress out here that the voting process then also has to consider a vote for the package's dependencies. This might become potentially a very recursive situation. Does it require a separate PR for all "leave packages" in the dependency tree or does the vote of the top-level package propagate through all dependencies?
I feel this would be a situation in which packages can easily be slipped through and marked as essential if not reviewed carefully in the dependency tree.

Reply:
I think that the clean solution is to say that when you vote on a package being included, you are also voting on its dependencies, except for those that are already on the list. It should be the responsibility of the proposer to provide a complete list of all required dependencies.